### ISSUE TITLE
9xd+ model name corrupt after initial eeprom conversion

### ISSUE THREAD

**Issue by jj-uk** [EdgeTX/edgetx#1854]
After loading ETX for the 1st time, the conversion from OTX 2.3.14 completed and the radio started up.

Some, not all, model names are corrupted. 
See the images. This shows the OTX and the ETX models.

**To Reproduce**
Fresh install ETX over the top of OTX.

**Expected behavior**
Model names to be preserved.

**Screenshots**
![image](https://user-images.githubusercontent.com/10943625/163855859-9af04053-642b-44c8-ba31-e3724f600698.png)

**General information**
Taken from version screen in radio:
 - FW: edgetx-9xd+
 - VERS: 2.7.0 (f79978b6)
 - DATE: 2022-04-16 03:41:02
 - CFGV: 221
 
- Hardware Frsky X9D+ 2015

**Comment by elecpower** [comment]
@jj-uk did you convert on radio or using Companion?
I loaded the zip into Ubuntu 20.04 builds of Companion 2.7 and 2.8 nightly and no corruptions

**Comment by jj-uk** [comment]
@elecpower I converted on the radio when prompted.

**Comment by elecpower** [comment]
@jj-uk thanks for the additional info

**Comment by driedeker** [comment]
Does the same on my X9D+se2019 includes some curuption in mixes

**Comment by driedeker** [comment]
Does the same on my X9D+se2019 includes some curuption in mixes etc

![corupt-flight-modes](https://user-images.githubusercontent.com/6246146/164108048-9ace0208-5b2e-4fa4-bd7d-1c4ed1d5f117.PNG)
![corupt-output-names](https://user-images.githubusercontent.com/6246146/164108062-a1da7793-3444-4a04-a0b0-faec1334deb6.PNG)
![corupt-mix-names](https://user-images.githubusercontent.com/6246146/164108073-547f05dd-e416-431f-b1ae-ff8d72f57040.PNG)

**Comment by alm-nl** [comment]
Confirmed on X9D Plus, coming from OpenTX 2.3.15, installing EdgeTX 2.8.5 and doing the conversion on the radio. I also installed 2.9.x, 2.10.x and 2.11.x where x is the current latest of each version, although I don't think that matters because 2.8.5 did the conversion. Did not use EdgeTX Companion yet.
It seems to happen with model names that are longer than 10 characters.

The active model also had this in the Outputs menu, I have checked other models and they don't seem to have this behavior. Edit: this was caused by outputs label names of 6 characters, other models did not have this and were therefore not affected.

**Comment by jj-uk** [comment]
I think EdgeTx dropped support for the x9d (2019) from v2.4, but still supports the x9d (2015) until v2.11 (?)

https://edgetx.org/supportedradios/#not-supported-by-the-edgetx-team-community-supported-only

OpenTx probably does almost everything you'd need on the x9d anyway.

**Comment by pfeerick** [comment]
No, it is supported as of 2.4, with 2.11 being the last version to support
that radio. However, it is only community supported since we have no
support from the manufacturer.
